docs/manifest.json

```diff
@@ -53,16 +53,28 @@
 		"markdown_source": "https:\/\/raw.githubusercontent.com\/WordPress\/gutenberg\/master\/docs\/extensibility.md",
 		"parent": null
 	},
+	{
+		"title": "Extending Blocks",
+		"slug": "extending-blocks",
+		"markdown_source": "https:\/\/raw.githubusercontent.com\/WordPress\/gutenberg\/master\/docs\/extensibility/extending-blocks.md",
+		"parent": "extensibility"
+	},
+	{
+		"title": "Plugins API",
```

Comment:
I'm hesitant here :) I wonder if this should leave in "Extensibility" or a separate section "modules/packages" listing all the READMEs of all available packages.

Reply:
Yes, I will remove. It's referenced in the overview document so should be fine for the time being.